src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -281,7 +280,30 @@ protected function describeContainerParameter($parameter, array $options = [])
 
     protected function describeContainerEnvVars(array $envs, array $options = [])
     {
-        throw new LogicException('Using the markdown format to debug environment variables is not supported.');
+        $this->write("# Environment Variables\n\n");
+
+        if ([] === $envs) {
+            $this->write("There are no environment variables\n");
+
+            return;
+        }
+
+        $this->write("NAME | PROCESSOR | DEFAULT | VALUE\n");
+        $this->write("---|---|---|---\n");
+
+        foreach ($envs as $env) {
+            $this->write(
+                sprintf(
+                    '%s | %s | %s | %s'.\PHP_EOL,
```

Comment:
This class uses only `\n` as newline character.
```suggestion
                    "%s | %s | %s | %s\n",
```